dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -328,6 +317,12 @@ void WebdatasetLoader::PrepareMetadataImpl() {
             output_indicies_.push_back(output);
             component.outputs.num++;
             was_output_set[output] = true;
+          } else {
+            std::call_once(multiple_files_single_component, [&]() {
```

Comment:
This will print it only once per WDS reader, not once per occurrence of the problem (or even per output). As a user attempting to fix that, I'd be deeply frustrated to fix the errors one by one. On the other hand, if the user is OK with that, it would be nice to have a way to limit or even silence these warnings.

Reply:
yeah but I don't think it's really a problem, it's just part of the requirements (and if it is a problem then it's really not that hard to deduce what other occurrences are, you just look at the composition of the tar file which you have to rebuild anyways if you wanna get rid of a file in the middle)

Reply:
OK.